tests/test_create_consensus_network.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/env/bin python3
+
+import unittest
+import filecmp
+import sys
+import os
+import shutil
+from SJARACNe.bin.create_consensus_network import create_consensus_network as cn
+from SJARACNe.bin.create_consensus_network import create_enhanced_consensus_network as ecn
+from SJARACNe.bin.create_consensus_network import uprob
+
+class TestConsensusNetwork(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        self.path = './test_answers'
+        if self.path is None:
+            os.mkdir('./test_answers')
+        cn('./tests/inputs/adjmat_dir', 0.05, './test_answers')
```

Comment:
Same as above, do not assume current work dir is writable. Use tempfile module instead to create temp directory.

Reply:
As above, implemented using tempfile to create a temporary directory